### PR TITLE
Resolve puppet-lint notices

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
puppet-lint flagged these lines

### CODE REVIEW

File: types/actions.pp [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-      Rsyslog::Actions::Outputs::Omusrmsg,
-    ]],
+      name          => String[1],
+      type          => Rsyslog::Modules::Output,
+      facility      => Optional[String[1]],
+      action_params => Optional[Rsyslog::Actions::Parameters],
+      queue_params  => Optional[Rsyslog::Queue::Parameters],
+      output_params => Optional[Variant[
+          Rsyslog::Actions::Outputs::Omamqp1,
+          Rsyslog::Actions::Outputs::Omelasticsearch,
```
Comment (igalic) [inline types/actions.pp:11]: we're currently having a discussion on our IRC channel why some of us don't like this change, while others are welcoming it

until that discussion is over, these changes should not be merged

**Review by igalic — COMMENTED** [review]
🙅🏻‍♀️

File: types/actions.pp (hunk 1 above)
Comment (dhollinger) [reply types/actions.pp:11]: @igalic Any idea how those discussions went?

Comment (ghoneycutt) [reply types/actions.pp:11]: Would be great if the discussion took place over the mailing list so asynchronous feedback could be given. If a project maintainer like @dhollinger does not know how the conversation went, that is not a good sign of transparency.

**Pull request closed without merging**